### DISCUSSION TITLE
Should we separate the "魔法部日志" from this repo?

### DISCUSSION THREAD

**Discussion by visualDust** [visualDust/ml.akasaki.space#81] [General]
Leaving this part here may result in an increase in the size of the repository

**Comment by Asthestarsfalll** [comment]
why?

**Reply by visualDust** [reply]
if we keep it here, the repo will be too big.

**Reply by visualDust** [reply]
I'm not thinking about move it myself, some one else did.

**Comment by BreezeShane** [comment]
It might become necessary one day. Well, why not move it before it's too big? 
To be honest, that  the part "魔法部日志" appears here could be a little inappropriate.... The site is mainly open for the green and.... maybe reading papers is too early for them.
The part aims at recording papers but this repo aims at creating a course of ML, so I agree to separate it.
Just my opinion......

**Comment by visualDust** [comment]
okay, sections were separated. see https://ml.akasaki.space/
